src/js/App/GlobalFilter/GlobalFilter.js

```diff
@@ -99,22 +99,12 @@ const GlobalFilter = () => {
 
   useEffect(() => {
     if (userLoaded && token && isAllowed()) {
-      if (!Object.values(selectedTags?.[workloads?.[0]?.name] || {})?.some(({ isSelected } = {}) => isSelected)) {
-        setValue({
-          ...(selectedTags || {}),
-          [workloads?.[0]?.name || 'Workloads']: {
-            ...selectedTags?.[workloads?.[0]?.name],
-            ...selectWorkloads(),
-          },
-        });
-      } else {
-        dispatch(globalFilterChange(selectedTags));
-      }
+      dispatch(globalFilterChange(selectedTags));
     }
   }, [selectedTags, isAllowed()]);
 
   useEffect(() => {
-    const sapTag = workloads?.[0]?.tags?.[1];
+    const sapTag = workloads?.[0]?.tags?.[0];
```

Comment:
I am just curious about this one. The SAP tag is always the first tag? And if so can this be refactored so it's a bit nicer on the eye?

Reply:
On the initial mocks, there was SAP and some other workload, hence the array. It's a static array passed from constants. I'm thinking about exporting just the SAP tag itself and bypass the workloads.

Reply:
Yeah, it would definitely make more sense. We can do it in a different PR though.